scripts/algebra-gas/README.md

```diff
@@ -0,0 +1,48 @@
+Scripts that help generate/evaluate gas parameters for generic algebra move module.
+
+## Quickstart guide
+Ensure you are on a machine with the [required spec](https://aptos.dev/nodes/validator-node/operator/node-requirements/).
+
+Ensure you have python3 and the following dependencies.
+```
+pip3 install numpy matplotlib
+```
+
+Ensure you `cd` to the repo root.
+
+Run the necessary benches.
+```
+cargo bench -p aptos-crypto -- hash/SHA2-256
+cargo bench -p aptos-crypto -- ark_bls12_381
+```
+
+If you don't have a `gas_per_ns`, estimate one using `hash/SHA2-256` bench results.
+```
+scripts/algebra-gas/load_bench_datapoints.py --bench_path target/criterion/hash/SHA2-256
+scripts/algebra-gas/fit_linear_model.py --dataset_path hash_SHA2_256.0-1025.json --plot
+```
+You will see some output like:
+```
+{"b": 336.51096106242346, "k": 4.868293006038344}
+```
+Take the `k`, and now you can do the calculation `gas_per_ns = 1000/k`.
```

Comment:
Where did 1000 come from?

Reply:
Fixed.

Reply:
Still don't know where this came from 🤣 ...

Reply:
Bump.

Reply:
updated and mentioned it's from the pre-defined sha256 formula.